src/cogent3/maths/stats/special.py

```diff
@@ -58,45 +62,28 @@ def one_minus_exp(x):
         return 1 - exp(x)
 
 
-def permutations(n, k):
-    """Returns the number of ways of choosing k items from n, in order.
-
-    Defined as n!/(n-k)!.
-    """
-    # Validation: k must be be between 0 and n (inclusive), and n must be >=0.
-    if k > n:
-        raise IndexError(f"Can't choose {k} items from {n}")
-    elif k < 0:
-        raise IndexError("Can't choose negative number of items")
-    elif n < 0:
-        raise IndexError("Can't choose from negative number of items")
-    if min(n, k) < 20 and isinstance(n, int) and isinstance(k, int):
-        return permutations_exact(n, k)
-    else:
-        return exp(ln_permutations(n, k))
+@c3warn.deprecated_callable(
+    version="2024.9", reason="use scipy.special.perm() instead", is_discontinued=True
+)
+def permutations(n, k):  # pragma: no cover
+    """being removed"""
+    return scipy.special.perm(n, k, exact=False)
 
 
-def permutations_exact(n, k):
-    """Calculates permutations by integer division.
+@c3warn.deprecated_callable(
+    version="2024.9", reason="use scipy.special.perm() instead", is_discontinued=True
+)
+def permutations_exact(n, k):  # pragma: no cover
+    """being removed"""
+    return scipy.special.perm(n, k, exact=True)
```

Comment:
The point here is not to replace our function with that from Scipy; rather, just use the decorator as indicated. Revert the change within the function to the original.